Documentation/network/clustermesh/aks-clustermesh-prep.rst

```diff
@@ -90,9 +90,9 @@ Install cluster one
     .. code-block:: bash
 
         cilium install \
-            --azure-resource-group "${AZURE_RESOURCE_GROUP}" \
-            --cluster-id 1 \
-            --config "cluster-pool-ipv4-cidr=10.10.0.0/16"
+            --set azure.resourceGroup="${AZURE_RESOURCE_GROUP}" \
```

Comment:
This `azure.resourceGroup` seems to be missed in the [Documentation/helm-values.rst](https://github.com/cilium/cilium/blob/main/Documentation/helm-values.rst)

Reply:
yeah some of these azure settings are not documented 😞 https://github.com/cilium/cilium/blob/c068a443049b5dedb34608069df4b1721b1e4669/install/kubernetes/cilium/values.yaml#L329-L335

Reply:
Can we create a follow-up issue for this?

Reply:
opened https://github.com/cilium/cilium/issues/26654